pkg/service/v0/data/user.go

```diff
@@ -7,6 +7,21 @@ type Users struct {
 
 // User holds the payload for a GetUser response
 type User struct {
+	// TODO needs better naming, clarify if we need a userid, a username or both
+	Enabled     string `json:"enabled" xml:"enabled"`
+	UserID      string `json:"id" xml:"id"`
+	Username    string `json:"username" xml:"username"`
+	DisplayName string `json:"display-name" xml:"display-name"`
+	Email       string `json:"email" xml:"email"`
+	Quota       *Quota `json:"quota" xml:"quota"`
+	UIDNumber   int64  `json:"uidnumber" xml:"uidnumber"`
+	GIDNumber   int64  `json:"gidnumber" xml:"gidnumber"`
+}
+
+// UserWithOtherFormat holds the payload for a GetUser response
+// when using the /cloud/users/{userid} endpoint
+// Currently the only difference is the DisplayName
+type UserWithOtherFormat struct {
```

Comment:
Could a more descriptive name help more than a comment here? 👼

Reply:
Yeah, I couldn't think of a fitting name though... :see_no_evil:.
Do you have something in mind?

Reply:
what about `GetUserResponse` since it's tied to the `GetUser` endpoint? But if [this comment](https://github.com/owncloud/ocis-ocs/pull/61#pullrequestreview-489475229) is correct and we can add both to the payload, then we shouldn't need this struct at all.

Reply:
Oh right, I missed that comment. That's an even better approach. Let's try that! :+1: